docs/source/deployment/deploy-model-locally.rst

```diff
@@ -77,7 +77,59 @@ CSV input must be a valid pandas.DataFrame CSV representation. For example:
 JSON Input
 **********
 
-JSON input must be a dictionary with exactly one of the following fields, further specifying the input data's type and encoding:
+You can either pass a flat dictionary corresponding to the desired model payload or wrap the
+payload in a dict with a dict key that specifies your payload format. 
+
+Raw Payload Dict
+^^^^^^^^^^^^^^^^
+
+If your payload is in a format that your mlflow served model will accept and it's in the supported
+models below, you can pass a raw payload dict.
+
+.. list-table::
+    :widths: 20 40 40
+    :header-rows: 1
+    :class: wrap-table
+
+    * - Supported Model Type
+      - Description
+      - Example
+    * - OpenAI Chat
```

Comment:
I'm afraid that "Model Type" is a bit vague concept (like it can be easily mistaken with flavor). Since we indeed have no limitation for models - it's just matter of customer's model accepts the Chat payload or not - probably we can replace it with sth like "supported request format"?

Reply:
Yeah this is tough because it feels like a one-off. Good clarification